src/sparsezoo/model/model.py

```diff
@@ -531,14 +562,14 @@ def _directory_from_files(
         display_name: Optional[str] = None,
         regex: Optional[bool] = False,
         allow_multiple_outputs: Optional[bool] = False,
-        **kwargs,
+        **kwargs: object,
```

Comment:
just curious, is this necessary?

Reply:
agreed, remove

Reply:
Actually they are used [here](https://github.com/neuralmagic/sparsezoo/blob/1dec9056b3fd931d4ffd611ceaf64ea793151755/src/sparsezoo/model/model.py#L100)